### PR TITLE
Add limitations

### PR DESCRIPTION
**Pull request by acarbonetto**

### CODE REVIEW

File: LIMITATIONS.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Known Limitations
+
+### Java Client Limitations
+
+- Limited JDK support outside of JDK 11 and above. Unable to build libraries on JDK 21.
+
+### Glide Core Limitations
+
+- Single slot transactions:  Transaction commands must be commited to a single slot, and multiple commands that require different slots will fail. See [#199](https://github.com/valkey-io/valkey-glide/issues/199).
```
Comment (Yury-Fridlyand) [inline LIMITATIONS.md:9]: Do we need an example there? Or for every item there?

File: LIMITATIONS.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- Function Load and server propagation:
+   - When loading functions to the server, glide will propagate the function library to existing cluster nodes only. Any new cluster nodes may not receive function libraries automatically.
+server nodes will not receive new libraries automatically.
+- Pub/Sub subscription commands: Subscribe commands require a new client and configuration. See: [PubSub documentation](https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#glide-pubsub-feature) for more information.
+- SCAN on cluster mode: Due to cluster layout, a scan request on a cluster of nodes cannot return a simple cursor. see: [SCAN documentation](https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#cluster-scan) for more information.
+- Large script requests are limited by the size of protobuf messages. see: [#1756](https://github.com/valkey-io/valkey-glide/issues/1756) for more information.
+- FCALL commands ignores the is_readonly_cmd flag and sends requests to replicas. see: [#1569](https://github.com/valkey-io/valkey-glide/issues/1569) for more information.
```
Comment (Yury-Fridlyand) [inline LIMITATIONS.md:16]: A bug not limitation I think

Comment (acarbonetto) [reply LIMITATIONS.md:16]: Yes.  It's okay to declare bugs here too.

Comment (Elen-Ghulam) [reply LIMITATIONS.md:16]: yes ok to list bugs

File: LIMITATIONS.md (hunk 1 above)
Comment (alexey-temnikov) [reply LIMITATIONS.md:9]: Let's add an inline example - what works, what doesn't  work

File: LIMITATIONS.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## Known Limitations
```
Comment (alexey-temnikov) [inline LIMITATIONS.md:1]: Aim - should be documented in the user friendly / developer friendly manner

File: LIMITATIONS.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+### Java Client Limitations
+
+- Limited JDK support outside of JDK 11 and above. Unable to build libraries on JDK 21.
+
+### Glide Core Limitations
```
Comment (raymond-lum) [inline LIMITATIONS.md:7]: Newbie won't know what Glide Core is. Would frame as "All Clients"

**Review by alexey-temnikov — CHANGES_REQUESTED** [review]
1. Add examples: what doesn't work, what works.
2. Explain impact of this limitation
3. Document workaround

File: LIMITATIONS.md (hunk 4 above)
Comment (asafpamzn) [reply LIMITATIONS.md:7]: should be part of the README

File: LIMITATIONS.md (hunk 1 above)
Comment (asafpamzn) [reply LIMITATIONS.md:9]: It is already documented. Please close the issue.

**Pull request closed without merging**